Typescript Building Fails

When building tsc fails with these errorrs.
```
$ tsc
node_modules/dank-twitch-irc/dist/mixins/roomstate-tracker.d.ts:11:47 - error TS2507: Type 'typeof EventEmitter' is not a constructor function type.

11 export declare class RoomStateTracker extends EventEmitter<RoomStateTrackerEvents> implements ClientMixin {
                                                 ~~~~~~~~~~~~

node_modules/dank-twitch-irc/dist/mixins/userstate-tracker.d.ts:13:47 - error TS2507: Type 'typeof EventEmitter' is not a constructor function type.

13 export declare class UserStateTracker extends EventEmitter<UserStateTrackerEvents> implements ClientMixin {
```

--- a/lib/mixins/roomstate-tracker.ts
+++ b/lib/mixins/roomstate-tracker.ts
@@ -1,5 +1,5 @@
 import * as debugLogger from "debug-logger";
-import * as EventEmitter from "eventemitter3";
+import { EventEmitter } from "eventemitter3";
 import { ChatClient } from "../client/client";
 import {
   hasAllStateTags,

--- a/lib/mixins/userstate-tracker.ts
+++ b/lib/mixins/userstate-tracker.ts
@@ -1,4 +1,4 @@
-import * as EventEmitter from "eventemitter3";
+import { EventEmitter } from "eventemitter3";
 import { ChatClient } from "../client/client";
 import {
   GlobalUserState,

